frontend-react/src/components/Admin/AdminReceiverDashboard.test.tsx

```diff
@@ -237,30 +235,28 @@ describe("AdminReceiverDashboard tests", () => {
     });
 
     test("sortStatusData and MainRender tests", async () => {
-        const { baseElement } = renderWithBase(
-            <MemoryRouter>
+        const { baseElement } = renderApp(
+            <NetworkErrorBoundary
+                fallbackComponent={() => <ErrorPage type="message" />}
+            >
                 <Suspense fallback={<></>}>
-                    <NetworkErrorBoundary
-                        fallbackComponent={() => <ErrorPage type="message" />}
-                    >
-                        {/*eslint-disable-next-line react/jsx-pascal-case*/}
-                        <_exportForTesting.MainRender
-                            datesRange={[
-                                new Date("2022-07-11"),
-                                new Date("2022-07-14"),
-                            ]}
-                            filterRowStatus={
-                                _exportForTesting.SuccessRate.ALL_SUCCESSFUL
-                            }
-                            filterErrorText={" "}
-                            filterRowReceiver={"-"}
-                            onDetailsClick={(
-                                _subdata: AdmConnStatusDataType[]
-                            ) => {}}
-                        />
-                    </NetworkErrorBoundary>
+                    {/*eslint-disable-next-line react/jsx-pascal-case*/}
```

Comment:
Blech, I might have already left a comment about this on your previous pull request, but we should get rid of the `exportForTesting` pattern in our codebase we encounter it/when we can -- if things need to be exported, they should just be exported.  While it's unavoidable in very rare cases, we shouldn't be formatting our source code around tests.  (Fortunately it's not too commonly used in this codebase, though!)

DEFINITELY DEFINITELY DEFINITELY out of the scope of this pull request -- just wanted to raise it for awareness as we touch files that have this pattern! FYI @penny-lischer @etanb